.travis.yml

```diff
@@ -41,4 +40,4 @@ script:
 
 after_success:
 
-  - ./ci_support/upload_or_check_non_existence.py ./recipe conda-forge --channel=pre
```

Comment:
Are you sure you wanted to merge this @Cadair? It doesn't appear to have gone wrong in https://anaconda.org/conda-forge/sunpy/files but I suspect that is because there was already an RC in the `pre` channel.